src/Dapr.Client/DaprClient.cs

```diff
@@ -210,6 +210,40 @@ public static CallInvoker CreateInvocationInvoker(string appId, string daprEndpo
             Dictionary<string, string> metadata,
             CancellationToken cancellationToken = default);
 
+        /// <summary>
+        /// // Bulk Publishes multiple events to the specified topic.
+        /// </summary>
+        /// <param name="pubsubName">The name of the pubsub component to use.</param>
+        /// <param name="topicName">The name of the topic the request should be published to.</param>
+        /// <param name="events">The list of events to be published.</param>
+        /// <param name="contentType">The content type of the event.</param>
+        /// <param name="cancellationToken">A <see cref="CancellationToken" /> that can be used to cancel the operation.</param>
+        /// <returns>A <see cref="Task" /> that will complete when the operation has completed.</returns>
+        public abstract Task<BulkPublishResponse> BulkPublishEventAsync<TValue>(
+            string pubsubName,
+            string topicName,
+            IReadOnlyList<TValue> events,
+            string contentType,
```

Comment:
Was this not included per event? Doesn't that imply that each event can be encoded separately?

Reply:
Per discussion offline, removed it.